pkgs/by-name/gp/gpt4all/package.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   config,
   stdenv,
+  gcc12Stdenv,
```

Comment:
how about in all-packages.nix  ?

`override { stdenv = gcc12Stdenv };`

also this should be resolved soon in nvcc

https://github.com/NVIDIA/nccl/issues/1743

leave a link to the issue in the override